.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
Yes, multi environment builds are a huge pain :(
If you don't feel like going through it, I think it should be fine to keep travis at py2+cv2, and test py3/cv3 locally - especially because that's the environment you'd be developing on, right?